.travis.yml

```diff
@@ -36,26 +40,28 @@ notifications:
 
 matrix:
   include:
+    # 2.7 build (until 0.14 is release)
     - os: linux
       python: 2.7
-      env: PIP_FLAGS="--pre"
+      env: WITH_PYAMG=1 QT=PyQt4
+      # Examples need strange versions of Matplotlib and Qt
+      # I don't think we should be increasing the minimum version requirement
```

Comment:
@hmaarrfk can you elaborate on this? Given [this comment](https://github.com/scikit-image/scikit-image/issues/3034#issuecomment-389424649), I'm inclined to increase minimum mpl to 2.0 (over a year old now), and minimum Qt to 5.

Reply:
Well, I guess that relates to the philosophy of what `scikit-image` is. Is it a plotting and visualization guide or an image processing library?

Personally, I don't think that matplotlib should even be a dependency. But I see where things like "viewers" and other things might be useful.

I'm ok with increasing minimum version requirements, but I feel like that is because I am inexperience and haven't had the (mis)forture of dealing with legacy software that cannot be updated.

Reply:
I think @efiring is right: it's hard to imagine a situation where you can't update MPL past 1.3 but you can update skimage to 0.14. In my view, conda environments have pretty much solved the update problem, because any user can install up-to-date dependencies in user-space.

More generally, we could have a calendar-based guide: we will support dependency versions up to three years old, for example.

In either view, 1.3 should be updated to either 1.5 or 2.0.

Regarding removing the dependency altogether... Well, now that we are doing separate dependencies for documentation/examples and the library, this is a possibility. But that's a big job, as we would need to extricate the viewer from the library. I think this is a worthwhile goal... For skimage 1.0. For now, let's just bump up the dependency. =D

Reply:
1.5 didn't work if i recall. I'll try 2.0. I could setup docker to test. But lets see how many commits it takes me.